src/components/ClaimReview/form/SelectUser.tsx

```diff
@@ -31,7 +31,7 @@ function SelectUser({ fetchOptions, ...props }) {
 
     return (
         <StyledSelect
-            mode="multiple"
+            mode={name === "usersId" ? "multiple" : "tags"}
```

Comment:
pass styled as a prop instead, the component shouldn't care about it's being for a user or a topic